addons/binding/org.openhab.binding.freebox/src/main/java/org/openhab/binding/freebox/handler/FreeboxHandler.java

```diff
@@ -175,15 +188,15 @@ private boolean authorize(boolean useHttps, String fqdn, String apiBaseUrl, Stri
             loginManager.setAppToken(configuration.appToken);
             loginManager.openSession();
             return true;
-        } catch (FreeboxException | InterruptedException e) {
-            logger.error("{}", e.getMessage());
+        } catch (Exception e) {
```

Comment:
It should imho not be the responsibility of this method to catch away any potential exceptions, but the truth is that if you don't catch them here the authorize will pass it on to the runnable which does not catch exceptions and it will terminate, so let's leave this one in place.